site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -92,90 +108,252 @@ Links might come and go on your page, and they're not the only way users can nav
 E.g., they may submit a form or even use an [image map].
 Your page might deal with these, but there's a long tail of possibilities which could just be simplified—something that the new Navigation API achieves.
 
+Additionally, the above doesn't handle back/forward navigation. There's another event for that, `"popstate"`.
```

Comment:
Events in this article seem to be without code font (e.g. lots of instances of "navigate", not `"navigate"`).

Reply:
I'll make them all use code font